src/Storage/StorageClient.php

```diff
@@ -167,20 +167,37 @@ public function bucket($name, $requesterPays = false)
      *           be either 'full' or 'noAcl'.
      *     @type string $fields Selector which will cause the response to only
      *           return the specified fields.
+     *     @type string $userProject The user project to be used for
```

Comment:
Update this and similar instances to:

If set, this is the ID of the project which will be billed for the request.